src/kata5.reachDestination.js

```diff
@@ -1,5 +1,9 @@
 const reachDestination = (distance, speed) => {
+const timeLeft = (distance/speed).toFixed(1)
+const roundTime = (Math.round(timeLeft*2))/2
+return roundTime
```

Comment:
You could just return `(Math.round(timeLeft * 2)) / 2` here - the variable assignment isn't necessary.